src/cljs/rems/actions/review.cljs

```diff
@@ -24,18 +24,18 @@
            :error-handler status-modal/common-error-handler!})
    {}))
 
-(def ^:private action-form-id "comment")
+(def ^:private action-form-id "review")
 
-(defn comment-action-button []
+(defn review-action-button []
   [action-button {:id action-form-id
                   :text (text :t.actions/comment)
                   :on-click #(rf/dispatch [::open-form])}])
 
-(defn comment-view
+(defn review-view
   [{:keys [comment on-set-comment on-send]}]
   [action-form-view action-form-id
    (text :t.actions/comment)
-   [[button-wrapper {:id "comment"
+   [[button-wrapper {:id "review-button"
                      :text (text :t.actions/comment)
```

Comment:
I guess you could also change the translations as part of the front-end changes.